modules/engage-paella-player/pom.xml

```diff
@@ -150,7 +161,7 @@
               <outputDirectory>${basedir}/target/classes/ui/</outputDirectory>
               <resources>
                 <resource>
-                  <directory>./target/gulp/paella-opencast</directory>
+                  <directory>./build/paella-opencast</directory>
```

Comment:
DCE checks in the Player build so the Player doesn't have to be rebuilt during a regular Opencast maven build.